src/app/functions/server/onerep.ts

```diff
@@ -38,7 +38,7 @@ export type ShowProfileResponse = CreateProfileRequest & {
   status: "active" | "inactive";
   created_at: ISO8601DateString;
   updated_at: ISO8601DateString;
-  url: `https://api.onerep.com/profiles/${number}`;
+  url: `${string}/profiles/${number}`;
```

Comment:
Does `${string}` actually work here, isn't this looking for the variable `string` which (I don't think) is set?

Reply:
I would expect something like an environment variable or a string containing the host name.

Reply:
Since it's in the definition of a type, and template literal types are statically resolved at compile time, I don't think I can place a variable in there. The previous URL links to the real website, but I wanted the URL to link to a mock profile. And string should be interpreted as a type, not a variable (I think). Let me know if its a mistake